src/dynarec/rv64/dynarec_rv64_f30f.c

```diff
@@ -153,6 +168,15 @@ uintptr_t dynarec64_F30F(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
             GETGXSS_empty(v1);
             FSQRTS(v1, v0);
             break;
+        case 0x53:
+            INST_NAME("RCPSS Gx, Ex");
+            nextop = F8;
+            GETEXSS(v0, 0);
+            GETGXSS_empty(v1);
+            LUI(x3, 0x3F800); // 1.0f
+            FMVWX(v1, x3);
```

Comment:
warning, v1 can be the same as v0, you might need a scratch regs here

Reply:
Indeed.

Reply:
Fixed